openblas/src/main/java/org/bytedeco/openblas/presets/openblas_nolapack.java

```diff
@@ -136,7 +139,8 @@ public class openblas_nolapack implements LoadEnabled, InfoMapper {
     public String[] functionsToSkip() {
```

Comment:
Please revert these changes that we don't need anymore.

Reply:
I left functionsToSikip() intentionally, as an option for future configuration. But, of course, if you think it's better to remove it, I'll do it.

Reply:
reverted.